cmd/mirror.go

```diff
@@ -27,18 +28,16 @@ import (
 	"github.com/pingcap/tiup/pkg/environment"
 	"github.com/pingcap/tiup/pkg/localdata"
 	"github.com/pingcap/tiup/pkg/repository"
-	"github.com/pingcap/tiup/pkg/repository/remote"
+	"github.com/pingcap/tiup/pkg/repository/model"
+	ru "github.com/pingcap/tiup/pkg/repository/utils"
+	"github.com/pingcap/tiup/pkg/repository/v0manifest"
```

Comment:
I think we can remove the `v0manifest` in the `v1.4.0`. @AstroProfundis

Reply:
I agree, https://github.com/pingcap/tiup/issues/906